rules/aws/credential_access_aws_iam_assume_role_brute_force.toml

```
@@ -0,0 +1,49 @@
+[metadata]
+creation_date = "2020/07/16"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/07/16"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a high number of failed attempts to assume an AWS IAM role. IAM roles are used to delegate access to users or
```

```suggestion
Identifies a high number of failed attempts to assume an AWS Identity and Access Management (IAM) role. IAM roles are used to delegate access to users or
```
it may be self explanatory, but I usually define the acronym the first time in the description, just for context, then use the abbreviation the rest of the way.

---

Perfect. Thanks